packages/klarna-integration/project.json

```diff
@@ -0,0 +1,23 @@
+{
+  "root": "packages/klarna-integration",
+  "sourceRoot": "packages/klarna-integration/src",
+  "projectType": "library",
+  "targets": {
+    "lint": {
+      "executor": "@nrwl/linter:eslint",
+      "outputs": ["{options.outputFile}"],
+      "options": {
+        "lintFilePatterns": ["packages/klarna-integration/**/*.{ts,tsx}"]
+      }
+    },
+    "test": {
+      "executor": "@nrwl/jest:jest",
+      "outputs": ["coverage/packages/klarna-integration"],
+      "options": {
+        "jestConfig": "packages/klarna-integration/jest.config.js",
+        "passWithNoTests": true
```

Comment:
Do you need this option for current package?

Reply:
Looks like yes. Without it CI checks are failing.

Reply:
If so, that menas there is no tests inside this package, but we do have a `test.tsx` here?
Can you please double check the root cause of that CI failure?